ert_shared/ensemble_evaluator/entity/ensemble_legacy.py

```diff
@@ -114,6 +151,8 @@ def _evaluate(self):
             self._allow_cancel.set()
             try:
                 asyncio.get_event_loop().run_until_complete(self._aggregate_future)
+                timeout_queue.put_nowait(None)
+                asyncio.get_event_loop().run_until_complete(send_timeout_future)
```

Comment:
Maybe do a small refactor whilst you're already here 🤣 

```suggestion
                async def _run_queue():
                    await self._job_queue.execute_queue_async(
                        dispatch_url,
                        self._ee_id,
                        threading.BoundedSemaphore(value=CONCURRENT_INTERNALIZATION),
                        queue_evaluators,
                        cert=cert,
                        token=token,
                    )
                    await timeout_queue.put(None)
                    await send_timeout_future
                asyncio.get_event_loop().run_until_complete(_run_queue())
```

You don't really need the `gather` anymore, and you can do `run_until_complete`. Idk, just a suggestion.